<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuProf</class>
 <widget class="QDialog" name="menuProf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>701</width>
     <height>341</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>341</height>
    </size>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_Notes">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Ajouter Notes</string>
    </attribute>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white</string>
     </property>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>381</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_25">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>CNE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="texte_CNE_Etudiant_2">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_26">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Id de la matière</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="texte_Id_Matier_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_27">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>La note</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="texte_Note_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Date d'evaluation</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="Date">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="accelerated">
            <bool>false</bool>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy </string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_28">
       <item>
        <widget class="QPushButton" name="pushButton_ajouter_6">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border:none;
color:white;
background-color:pink;
font: 100 10pt “Arial”;
border-radius:20px solid;
}
QPushButton:Hover
{
background-color:gold;
border-radius:20px;
border-radius:30px;

}
QPushButton:Hover:pressed
{
background-color:green;
color:blue
border-bottom:10px solid black;
}

</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_supprimer_6">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border:none;
color:white;
background-color:tan;
font: 100 10pt “Arial”;
border-radius:20px solid;
}
QPushButton:Hover
{
background-color:gold;
border-radius:20px;
border-radius:30px;

}
QPushButton:Hover:pressed
{
background-color:green;
color:blue
border-bottom:10px solid black;
}

</string>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_afficher_6">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border:none;
color:white;
background-color:pink;
font: 100 10pt “Arial”;
border-radius:20px solid;
}
QPushButton:Hover
{
background-color:gold;
border-radius:20px;
border-radius:30px;

}
QPushButton:Hover:pressed
{
background-color:green;
color:blue
border-bottom:10px solid black;
}

</string>
         </property>
         <property name="text">
          <string>Afficher </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>201</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../Screenshot 2024-01-08 145938.png</pixmap>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>531</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>541</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>360</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_30">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_All_Etudiant_8">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border:none;
color:white;
background-color:pink;
font: 100 10pt “Arial”;
border-radius:20px solid;
}
QPushButton:Hover
{
background-color:gold;
border-radius:20px;
border-radius:30px;

}
QPushButton:Hover:pressed
{
background-color:green;
color:blue
border-bottom:10px solid black;
}

</string>
      </property>
      <property name="text">
       <string>Etudiants</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_All_Matiers_8">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border:none;
color:white;
background-color:tan;
font: 100 10pt “Arial”;
border-radius:20px solid;
}
QPushButton:Hover
{
background-color:gold;
border-radius:20px;
border-radius:30px;

}
QPushButton:Hover:pressed
{
background-color:green;
color:blue
border-bottom:10px solid black;
}

</string>
      </property>
      <property name="text">
       <string>Matieres</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
